experimental/packages/opentelemetry-instrumentation-grpc/src/grpc-js/serverUtils.ts

```diff
@@ -159,29 +159,29 @@ export function handleServerFunction<RequestType, ResponseType>(
   callback: SendUnaryDataCallback<unknown>
 ): void {
   switch (type) {
-    case 'unary':
-    case 'clientStream':
-    case 'client_stream':
-      return clientStreamAndUnaryHandler(
-        span,
-        call,
-        callback,
+  case 'unary':
```

Comment:
What about adding a [`SwitchCase` option](https://eslint.org/docs/rules/indent#switchcase) to indent the `case` clause as we had before?

Reply:
Like it :)
Added and updated the files accordingly 👍